typescript/infra/fork.sh

```diff
@@ -1,6 +1,7 @@
 ENVIRONMENT=$1
 MODULE=$2
 CHAIN=$3
+WARP_ROUTE_ID=$4
 
```

Comment:
Support warp route ID

Reply:
just wanna confirm that if you're running this for e.g. core, the cli arg parsing still works / is fine with just receiving `--warpRouteId`

Reply:
Addresse!